scene/3d/physics_body.cpp

```diff
@@ -791,6 +812,14 @@ void RigidBody::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_weight", "weight"), &RigidBody::set_weight);
 	ClassDB::bind_method(D_METHOD("get_weight"), &RigidBody::get_weight);
 
```

Comment:
Just getter methods for now. I wish I could turn them into read-only properties but I can see that properties are saved even though I don't specify any setter